src/commands/vote.ts

```diff
@@ -0,0 +1,18 @@
+import { FireMessage } from "../../lib/extensions/message";
+import { Language } from "../../lib/util/language";
+import { Command } from "../../lib/util/command";
+
+export default class extends Command {
+  constructor() {
+    super("vote", {
+      description: (language: Language) =>
+        language.get("VOTE_COMMAND_DESCRIPTION"),
+      clientPermissions: ["SEND_MESSAGES"],
+      aliases: ["dbl", "top.gg", "dboats", "discord.boats"],
+    });
+  }
+
+  async exec(message: FireMessage, args: { name: string }) {
+    await message.channel.send("https://fire-is-the.best/vote");
```

Comment:
```suggestion
    await message.channel.send("https://fire-is-the.best/vote");
```
A constant should probably be used for this URL

Reply:
as in (const voteUrl: string = "https://fire-is-the.best/vote")

Reply:
> as in (const voteUrl: string = "https://fire-is-the.best/vote")

No, see [/lib/util/constants.ts](https://github.com/FireDiscordBot/bot/blob/rewrite/typescript/lib/util/constants.ts)